prover/crates/bin/circuit_prover/src/main.rs

```diff
@@ -85,6 +86,16 @@ async fn main() -> anyhow::Result<()> {
         ));
     }
 
+    // Snarkify
+    let cfg = CloudProverConfig {
```

Comment:
nit: get the configs from env var? and if not specified, make them default.

You can reference some default values here: https://github.com/snarkify/scroll-proving-agent/blob/main/charts/scroll-proving-agent/values/production.yaml